src/main/java/fdv/controller/PlaceKindController.java

```diff
@@ -0,0 +1,11 @@
+package fdv.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Controller;
+
+
+@Controller
+@AllArgsConstructor
+public class PlaceKindController {
```

Comment:
Read about [YAGNI](https://ru.wikipedia.org/wiki/YAGNI) principal, don't add useless class in pull request for future purposes. Add at exactly time when you will implement it.